src/opera/test/data/test_rtc_s1_config.yaml

```diff
@@ -0,0 +1,174 @@
+RunConfig:
+    Name: OPERA-RTC-S1-PGE-TEST-CONFIG
+
+    Groups:
+        PGE:
+            PGENameGroup:
+                PGEName: RTC_S1_PGE
+
+            InputFilesGroup:
+                InputFilePaths:
+                    - rtc_s1_test/input_dir/SAFE.zip
+                    - rtc_s1_test/input_dir/ORBIT.EOF
+
+            DynamicAncillaryFilesGroup:
+                AncillaryFileMap:
+                    dem_file: rtc_s1_test/input_dir/dem.tif
+
+            ProductPathGroup:
+                OutputProductPath: rtc_s1_test/output_dir
+                ScratchPath: rtc_s1_test/scratch_dir
+
+            PrimaryExecutable:
+                ProductIdentifier: RTC_S1
```

Comment:
Will we be adding ProductVersion here as in the other PGEs?

Reply:
Yes, I will likely merge your branch first, then update this branch to incorporate the new fields